services/scabbard/libscabbard/src/service/state.rs

```diff
@@ -162,6 +164,45 @@ impl ScabbardState {
         // initialize committed_batches metric
         counter!("splinter.scabbard.committed_batches", 0);
 
+        #[cfg(feature = "diesel-receipt-store")]
+        let receipt_store: Arc<RwLock<dyn ReceiptStore>> =
```

Comment:
Should this be an alternative to the existing lmdb files?

Reply:
Modified ScabbardState to only create the lmdb receipt store if "diesel-receipt-store" is not enabled